### PR TITLE
Feature/starting over

### PR DESCRIPTION
**Pull request by slashformotion**
- Keep the same API (almost, some functions were renamed)
- Remove old code and code specific to badaas

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module github.com/ditrit/verdeter
```
Comment (Zorin95670) [inline go.mod:1]: put this commit before code commit :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,212 @@
...
+    ```go
+    viper.SetDefault("time", 1661957668)
+    ```
+
+
+7. type default (0 for an integer)
+
+    If a key is *not set* and *not marked as required (using `(*VerdeterCommand).SetRequired(<key_name>)`)*, then a call to `viper.Get<Type>(<key_name>)` will return the default value for this `<Type>`.
+
+    *Example:* let's say thay we **did not** call (*VerdeterCommand).SetRequired("time")` to set the key "time" as required.
```
Comment (Zorin95670) [inline README.md:93]: missing ` before (*VerdeterCommand).SetRequired("time")

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Comment by slashformotion** [comment]
qsd

File: io.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package verdeter
+
+import (
+	"os"
+)
+
+// Return true if the file exists
+func FileExist(path string) bool {
```
Comment (Zorin95670) [inline io.go:8]: missing unit test

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**